packages/@aws-cdk/aws-applicationautoscaling/test/step-scaling-policy.test.ts

```diff
@@ -227,6 +227,43 @@ describe('step scaling policy', () => {
 
 
   });
+
+  test('step scaling with evaluation period & data points to alarm configured', () => {
```

Comment:
Once the `dataPointsToAlarm` verification is added, can we add a test to verify that the error is correctly thrown?